### ISSUE TITLE
Color contrast issues

### ISSUE THREAD

**Issue by mgifford** [ehrbase/ehrbase#1552]
### Before reporting an issue

- [x] I have searched existing issues
- [x] I have reproduced the issue with the latest release

### Environment information

The test is done on the sandbox https://sandkiste.ehrbase.org/

### Steps to reproduce

Login and go here:
https://sandkiste.ehrbase.org/composition

Run https://accessibilityinsights.io

Title: WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (vaadin-button[last-start-child=""],span[part="label"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Compositions - https://sandkiste.ehrbase.org/composition

Element path: vaadin-horizontal-layout:nth-child(1) > vaadin-button[last-start-child=""],span[part="label"]

Snippet: <span part="label"> <slot></slot> </span>

Related paths: 
  vaadin-horizontal-layout:nth-child(1) > vaadin-button[last-start-child=""]
  html

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.32 (foreground color: #e3163c, background color: #f4f5f7, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 141.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Title: WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (ace-widget,.ace_emptyMessage)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Compositions - https://sandkiste.ehrbase.org/composition

Element path: ace-widget,.ace_emptyMessage

Snippet: <div class="ace_emptyMessage" style="padding: 0px 9px; position: absolute; z-index: 9; opacity: 0.5;">Enter a composition</div>

Related paths: 
  ace-widget,.ace_scroller

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.5 (foreground color: #717474, background color: #1d1f21, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 141.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

<img width="1001" height="657" alt="Screenshot of errors" src="https://github.com/user-attachments/assets/6d8abbb3-2876-4bf3-95b3-c30ae5d88465" />

### Expected behavior

Colors should meet WCAG 2.2 AA as per the EAA and WAD (or Section 508 in the USA).

### Actual result

There isn't sufficient contrast according to WCAG.

**Comment by vmueller-vg** [comment]
@birgerhaarbrandt fyi

**Comment by vidi42** [comment]
@mgifford thank you your feedback. As I explained here https://github.com/ehrbase/ehrbase/issues/1553#issuecomment-3424910695, this repository doesn't host the sandbox environment UI. We maybe need to make that clearer.

I will now close this ticket, but know that I have forwarded your feedback internally.
